### PR TITLE
Added image and input component

### PR DESCRIPTION
**Pull request by TusharJade**
### Added Input and Image

Screenshot (18)
e8223b26-e227-40fc-9409-ae8861cfbc93.png)
Screenshot (19)

Screenshot (17)

### CODE REVIEW

**Comment by TusharJade** [comment]
### Added Input and Image

File: Components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <li class="nav-link"><a class="nav-anchor active" href="/Components/Image/image.html">Image</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Input/input.html">Input</a></li>
+        </ul>
+    </nav>
+
+    <section class="main-container">
+        <h1 class="main-heading">Image</h1>
+        <div class="separator"></div>
+        <h3 class="sub-heading">Responsive Image</h3>
+   <p class="des-para">Add img-responsive class to img tag. Image will grow to the 100% width of it's parent container.</p>
```
Comment (sakshi006) [inline Components/Image/image.html:37]: Maybe take an image that covers the entire container.
Right now, a scroll bar is appearing mostly due to the iframe not being responsive.
Screenshot (532)

**Review by sakshi006 — COMMENTED** [review]
Other than image, good to go.

**Comment by Balakumar3018** [comment]
Great work.
Suggestions - Make iframe responsive

**Comment by TusharJade** [comment]
> Great work. Suggestions - Make iframe responsive

yes forgot to look at responsiveness will fix it now

File: Components/Image/image.html (hunk 1 above)
Comment (TusharJade) [reply Components/Image/image.html:37]: Yahh will make iframe responsive then I think issue will solve

Comment (TusharJade) [reply Components/Image/image.html:37]: Got it will use image which use full size

**Comment by TusharJade** [comment]
> > Great work. Suggestions - Make iframe responsive
> 
> Made iframes responsive have a look.

File: Components/Image/image.html (hunk 1 above)
Comment (TusharJade) [reply Components/Image/image.html:37]: Made that image responsive have a look

**Pull request merged**